lib/utils/properties-panel.js

```diff
@@ -298,10 +302,6 @@ export function getErrorMessage(id, report) {
     return 'Cannot be an expression.';
   }
 
-  if (data.type === ERROR_TYPES.SECRET_EXPRESSION_FORMAT_DEPRECATED) {
-    return 'Secret expression format deprecated.';
```

Comment:
Dropped and will be re-added?

Reply:
Since we don't want to show an error in the properties panel this code is useless.